StreetDrop/StreetDrop/Application/SceneDelegate.swift

```diff
@@ -73,25 +73,20 @@ private extension SceneDelegate {
     }
     
     func handlingSharedMusic(isLaunched: Bool, components: URLComponents) {
-        guard let query = components.query,
-              let decodedParams = query.fromBase64SafeURL()
-        else { return }
-        
-        print(decodedParams)
-        let items = decodedParams.split(separator: "=")
-        guard items.count > 1 else { return }
-        
-        if items.first == "itemID" {
-            if let itemID = Int(items[1]) {
-                if isLaunched == false {
-                    navigateToCommunity(with: itemID)
-                } else {
-                    UserDefaults.standard.set(itemID, forKey: UserDefaultKey.sharedMusicItemID)
-                }
-            } else {
-                print("itemID missing")
+        let items = components.queryItems ?? []
+                for item in items {
```

Comment:
```suggestion
        for item in items {
            if (item.name == "itemID".base64UrlEncode()){
                if let itemID = item.value?.base64UrlDecode(), 
                    let decodedItemID = Int(itemID) {
                    if isLaunched == false {
                        navigateToCommunity(with: decodedItemID)
                    } else {
                        UserDefaults.standard.set(decodedItemID, forKey: UserDefaultKey.sharedMusicItemID)
                    }
                } else {
                    print("itemID missing")
                }
            }
        }
```
줄 간격 및 넘김 처리가 되었으면 좋겠네요~~

Reply:
수정했습니다.